mindmeld/cli.py

```diff
@@ -48,6 +48,15 @@
 
 CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"], "auto_envvar_prefix": "MM"}
 
+# deprecation warning for python 3.5
+if sys.version_info < (3, 6):
+    deprecation_msg = (
+        "DEPRECATION: Python 3.5 reached end of life on 13 Sept 2020. MindMeld will deprecate"
+        " official support for Python 3.5 in the next release. Please consider migrating"
+        " your application to Python 3.6 and above."
```

Comment:
Ideally we would support the latest python (3.8) and encourage them to use that. At least I'd like to recommend 3.7 BUT we don't have continuous integration set up for those so this is probable best as is (3.6).